src/Server/Workspace.php

```diff
@@ -200,4 +214,75 @@ public function xdependencies(): array
         }
         return $dependencyReferences;
     }
+
+    /**
+     * Fires when client changes settings in the client
```

Comment:
Change this to the protocol wording:

> A notification sent from the client to the server to signal the change of configuration settings.

https://github.com/Microsoft/language-server-protocol/blob/master/protocol.md#didchangeconfiguration-notification